feat(connectors): add google sheets connector docs (#2178)
camunda/camunda-platform-docs: docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md

- [GitLab Connector](/components/connectors/out-of-the-box-connectors/gitlab.md) - Manage [GitLab](https://about.gitlab.com/) issues and releases from your BPMN process.
- [Google Drive Connector](/components/connectors/out-of-the-box-connectors/googledrive.md) - Create folders or files from a [Google Drive](https://www.google.com/drive/) template from your BPMN process.
- [Google Maps Platform Connector](/components/connectors/out-of-the-box-connectors/google-maps-platform.md) - Validate addresses, retrieve postal addresses, and calculate distances with [Google Maps Platform Service](https://mapsplatform.google.com/) from your BPMN process

> - [Google Sheets Connector](docs/components/connectors/out-of-the-box-connectors/google-sheets.md) - allows you to work with an existing or new empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.

Reviewer: ```suggestion
- [Google Sheets Connector](/components/connectors/out-of-the-box-connectors/google-sheets.md) - allows you to work with an existing or new empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```

> - [Google Sheets Connector](docs/components/connectors/out-of-the-box-connectors/google-sheets.md) - Create empty spreadsheet on [Google Drive](https://drive.google.com/) and work with it or already existing one from your BPMN process.

Reviewer: ```suggestion
- [Google Sheets Connector](docs/components/connectors/out-of-the-box-connectors/google-sheets.md) - Work with a new or empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```

Author: changed
